nautobot/docs/development/core/model-checklist.md

```diff
@@ -0,0 +1,139 @@
+# Model Development Checklist
+
+A general best-practices checklist to follow when adding a net-new data model in Nautobot or adding new fields to an existing model.
+
+## Bootstrapping a new Model
+
+- ▢ Implement model in `nautobot.<app>.models` module
```

Comment:
Why the `▢` characters? To make it look like a checklist?

Reply:
That was the goal, yes. I'd initially done `[ ]` to match the GitHub format but it doesn't look great in the rendered docs.

Reply:
Another option *might* be some custom CSS to set the list bullet style to use `▢`?

Reply:
That might be nice to get rid of the bullets and only have the boxes